### PR TITLE
Versioned Api

### PR DESCRIPTION
**Pull request by acnagrellos**

### CODE REVIEW

File: 01 Api Versioned/src/Api.Features/Versioned/Api.Versioned/VersionConstants.cs [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+            if (!string.IsNullOrWhiteSpace(configValue))
+            {
+                try
+                {
+                    return (T)TypeDescriptor.GetConverter(typeof(T)).ConvertFromInvariantString(configValue);
+                }
+                catch (Exception)
+                {
+
+                }
```
Comment (franmolmedo) [inline 01 Api Versioned/src/Api.Features/Versioned/Api.Versioned/VersionConstants.cs:41]: Nothing here?

File: 01 Api Versioned/src/Api.Features/Versioned/Api.Versioned/VersionConstants.cs [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        public const string ConfVersionHeader = "configuration:api-header";
+        public const string ConfVersionDefault = "configuration:api-version-default";
+
+        //
+        // Versioned Constants
+        //
+        public const string VersionHeader = "api-header";
+        public const int VersionDefault = 1;
+
+        public static T GetSettingOrDefaultValue<T>(string settingKey)
```
Comment (franmolmedo) [inline 01 Api Versioned/src/Api.Features/Versioned/Api.Versioned/VersionConstants.cs:28]: Useless

File: 01 Api Versioned/test/TestRunner/ApiTestRunner.Core/Controllers/VersionController.cs [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    [RoutePrefix(Constants.BaseRoute + "/" + Constants.Version)]
+    public class VersionController : BaseController
+    {
+        [HttpGet]
+        [ResponseType(typeof(bool))]
+        [VersionedRoute(Constants.GetTest, VersionConstants.VersionDefault)]
+        public async Task<IHttpActionResult> GetTestBoolean()
+        {
+            return await Task.FromResult(Ok("Esta es la versión 1"));
+        }
```
Comment (franmolmedo) [inline 01 Api Versioned/test/TestRunner/ApiTestRunner.Core/Controllers/VersionController.cs]: English pls

File: 01 Api Versioned/src/Api.Features/Versioned/Api.Versioned/VersionConstants.cs (hunk 1 above)
Comment (acnagrellos) [reply 01 Api Versioned/src/Api.Features/Versioned/Api.Versioned/VersionConstants.cs:41]: If catch an exception the code return the default value in next lines.

File: 01 Api Versioned/src/Api.Features/Versioned/Api.Versioned/VersionConstants.cs (hunk 2 above)
Comment (acnagrellos) [reply 01 Api Versioned/src/Api.Features/Versioned/Api.Versioned/VersionConstants.cs:28]: I don´t agree. But maybe I need an infrastructure project for this.

**Pull request merged**